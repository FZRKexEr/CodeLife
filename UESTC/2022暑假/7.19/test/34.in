10
twuefyjbespxamvozqucldopcubpovabjgmnfqnsztdvpoawvxusurstmutjwmaixsriiduolauoiglnofhnimipnpmrxrvatfgb
9
mljzarbesxvquludbiizwbeocxjmwhhgesegxzwdlxmokqycssjigpvclugkknazloqthpkmrypdbdsrktduhgbmcohhiprrmxvd
9
hzpybxvmwjkgcnrubtmhpocknhutuhdyqtiyghmsxzbqtlhpyqtkbvisdqewbzufrtsisyaiytpjwsjnormyrwjvtbgmbevpwrcm
7
geituduhokiddyjvkvyzvgkbapgugotdyklklnxcxbnhnzpvongjhohgundehajzszjketdkzjhydtqsslsyjyurucqsnmtyhmdl
9
zmwrbmouabplvengbtyvdjvxyrxpmxvkxesooncfzwhymjqyifvrzcipztpraisajjfdolktyifyugjzexqicunxzcjretjfcvbf
8
eofuutvmekiftklicadjkaheoihxehrdgoiesclksekqzmkygnbdxdxhmtjfowbtxtzzuuaouaktfgxogsdiazjflojvobtxshio
7
unqdoxkynrehkpsogdkylvklpazpfmnakrqnyezzjbiylsmncmhatojyeyhmmiijlnfvweoymddojykaxfbtjhgxininsptuhora
3
dykpqqequbiyddboqmnjfqapxzeuhjevvfobszhkddyvtxnqfbfuukuuhqldddzuamtbtqfwytrmcmcbtepjyxphpudqrlwbohlm
10
mkdadaycccvqoqadenbwvlonuhiswcmmqmaxlxmecpsutnaophmifcyomjjadorlaucfomtsqatqhxakiuvtytluxcpgrasrpoqx
5
qbamhyxppzqpyviwsxqkdgdywsghclxpxpldgyhniapwjebferxwzquruhrfeoymmypxikmzcifzejcmpctgtepiscfxyzrzcwll
4
