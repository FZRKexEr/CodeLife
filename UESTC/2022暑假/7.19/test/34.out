2048
2048
16384
2048
4096
16384
603979768
1024
1048576
33554432
